content/blog/2020-05-14-may-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,180 @@
+  extremely sharp guide to practical steps anyone can take to improve the
+  reproducibility of their ML projects. She really nails the complexity of the
+  workflow and the importance of decoupling code and data (which we obviously
+  agree with very much 😏). The graphics are also 💯- Tania is a developer
+  advocate to follow.
+
+<external-link
+href="https://dev.to/azure/10-top-tips-for-reproducible-machine-learning-36g0"
+title="10 top tips for reproducible Machine Learning"
+description="The one where you get some advice to make your workflows more reproducible"
+link="dev.to"
+image="/uploads/images/2020-05-14/dev_logo.png"/>
+
+- [Vimarsh Karbhari](https://medium.com/@vimarshk) blogged about how teams that
+  work with data can strategize better about versioning their data and analysis
+  pipelines. On the opposite end of giving very practical recommendations,
+  Vimarsh stresses a deliberate and caeful approach. He emphasizes how the
+  team's choices should depend on factors like project maturity and how much
+  flexibility is going to be needed. It's a solid overview of how to begin
+  thinking about MLOps at a high level.
+
+<external-link
+href="https://medium.com/acing-ai/ml-ops-data-science-version-control-5935c49d1b76"
+title="ML Ops: Data Science Version Control"
+description="Data versioning primer for model, data and code."
+link="medium.com"
+image="/uploads/images/2020-05-14/acing_ai.png"/>
+
+- Over at [AutoRegresed](https://www.autoregressed.com/), Jack Pitts shared a
+  thorough tutorial about using [Pipenv](https://pypi.org/project/pipenv/), DVC
+  and Git together. As a trio, this manages dependencies and versions the
+  working environment, source code, dataset _and_ trained models. It's not only
+  a cool use case, but a very clear step-by-step explanation that should be easy
+  to try at home. Stay till the end for a neat trick about deploying a model as
+  a web service with Pipenv and DVC.
+
+<external-link
+href="https://www.autoregressed.com/blog/pipenv-and-dvc-reproducibility-in-data-science"
+title="Pipenv and DVC: Reproducibility in Data Science"
+description="Without standards and tools to easily reproduce models, Data Science teams can become bogged down in technical debt that will make it difficult to deploy and iterate on models. "
+link="autoregressed.com"
+image="/uploads/images/2020-05-14/ar_logo.jpg"/>
+
+## Nice tweets
+
+Last, here are some of our favorite tweets to read this past month:
+
+https://twitter.com/braaannigan/status/1257918525345234949
+
+https://twitter.com/josh_wills/status/1249774857614553097
+
+https://twitter.com/tcgarvin/status/1258855168436813826
+
+_Thank you, thank you very much._
+
+<iframe src="https://giphy.com/embed/gJ2sDSKAQHUCIYUFhx" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/TheWiggles-wiggles-the-thewiggles-gJ2sDSKAQHUCIYUFhx">via GIPHY</a></p>
```

Comment:
so, are we changing this? (probably, didn't get your answer - was it a confirmation or not :) )

Reply:
ah yes, missed this one! will be changing it to just the link, which embeds like a twitter link